src/node/rpc/endpoint.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/json.h"
+#include "kv/map.h"
+
+#include <string>
+#include <utility>
+
+namespace ccf
+{
+  namespace endpoints
+  {
+    using URI = std::string;
+
+    struct EndpointKey
+    {
+      URI method;
```

Comment:
I think we should avoid the term method here if we can (I know we've used it historically because of JSON RPC), perhaps `uri_suffix` or something?